### PR TITLE
Vale and markdownlint fixes for docs/community

### PR DESCRIPTION
**Pull request by cgcardona**
* Add accepted words to `accept.txt`
* Vale and markdownlint updates for the `docs/community` directory
* Add `false` markdownlint config for `MD024` - (Multiple headers with same content) should only trigger under the same nesting
* Add `false` markdownlint config for `MD036` - sees emphasized paragraphs as header

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: errata-ai/vale-action@reviewdog
         with:
-          files: '["docs/overview","docs/subnets", "README.md", "style-checker-notes.md", "style-guide.md"]'
+          files: '["docs/build", "docs/community", "docs/overview","docs/subnets", "README.md", "style-checker-notes.md", "style-guide.md"]'
```
Comment (yulin-dong) [inline .github/workflows/ci.yml:31]: I don't see a path called `docs/build` on https://github.com/ava-labs/avalanche-docs/tree/master/docs. Please remove it.

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (cgcardona) [reply .github/workflows/ci.yml:31]: There are no errors in the docs/build directory. 

```zsh
vale docs/build
✔ 0 errors, 0 warnings and 0 suggestions in 0 files.
```

Would you like like me to create a new PR w/ just that single addition to the vale files?

**Comment by cgcardona** [comment]
All of the vale and markdownlint checks are now passing for the `docs/community` directory.

```zsh
➜  avalanche-docs git:(vale-fixes-for-docs-community) ✗ markdownlint docs/community
➜  avalanche-docs git:(vale-fixes-for-docs-community) ✗ vale --minAlertLevel=error docs/community
✔ 0 errors, 0 warnings and 0 suggestions in 58 files.
```

File: .github/styles/Vocab/Products/accept.txt [hunk 2]
```diff
@@ -1,44 +1,81 @@
...
 APIs
 architecting
+Arweave
+auctionId
 auditable
 autogenerated
 Ava
 Ava Labs
 Avalanche-CLI
+avalanchego
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:25]: Is this needed? Where do we need all lower-case `avalanchego`?

File: .github/styles/Vocab/Products/accept.txt [hunk 3]
```diff
@@ -1,44 +1,81 @@
...
 AvalancheGo
 Avascan
+avax
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:28]: same with avax, where do we need all lower-case avax? cc: @dasconnor

File: .github/styles/Vocab/Products/accept.txt [hunk 4]
```diff
@@ -1,44 +1,81 @@
...
+Avax
 AVAX
+Berkay
+Binance
+bintools
+Blanco
+blackhole
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:35]: I think this had been fixed by another PR merged recently.

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: .github/styles/Vocab/Products/accept.txt [hunk 5]
```diff
@@ -1,44 +1,81 @@
+721s
 ABIs
 addValidator
 AddValidator
+Agrawal
+aggresive
+Aggresive
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:7]: these are wrong spelling, right? it should 2 s in aggressive.

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: .github/styles/Vocab/Products/accept.txt (hunk 3 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:28]: It's in some URLs. Such as the link at the top of this file https://github.com/ava-labs/avalanche-docs/tree/community-provider-section/docs/community

File: .github/styles/Vocab/Products/accept.txt (hunk 2 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:25]: It's also in some URLs.

Comment (yulin-dong) [reply .github/styles/Vocab/Products/accept.txt:25]: I think if you put URL into ``, they should be ignored. @dasconnor is my understanding correct? How to ignore avalanchego inside a URL?

File: .github/styles/Vocab/Products/accept.txt [hunk 6]
```diff
@@ -1,44 +1,81 @@
...
+blockNumber
 blockTimestamp
 bool
+bridgeAdmin
+Brunain
+BUIDLing
 calldata
 CAPTCHA
 ChainBridge
+chainlink
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:49]: Is there a case for all lower-case chainlink?

File: .github/styles/Vocab/Products/accept.txt [hunk 7]
```diff
@@ -1,44 +1,81 @@
...
 Coreth
 Crabada
+crypto
+cryptocurrencies
+cryptocurrency
+Ctrl
+Daly
+DAO
 dapp
+Dapp
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:74]: @dasconnor what is your opinion on dapp/Dapp?

File: .github/styles/Vocab/Products/accept.txt [hunk 8]
```diff
@@ -47,49 +84,112 @@ dev
 Dexalot
 Docusaurus
 dotenv
+drizzleOptions
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:87]: What is this?

File: .github/styles/Vocab/Products/accept.txt [hunk 9]
```diff
@@ -47,49 +84,112 @@ dev
...
+Enums
+env
 EOAs
+ERC
+ERC721s
 ESLint
+Eteng
 Ethereum
+Ethers
+ethtx
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:98]: eth tx?

File: .github/styles/Vocab/Products/accept.txt [hunk 10]
```diff
@@ -47,49 +84,112 @@ dev
...
+eVoting
+expectRevert
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:100]: This should not be here.

File: .github/styles/Vocab/Products/accept.txt [hunk 11]
```diff
@@ -47,49 +84,112 @@ dev
...
 feeRecipient
+finney
+fuji
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:103]: I think all reference to Fuji should be Fuji with first letter upper-cased.

File: .github/styles/Vocab/Products/accept.txt [hunk 12]
```diff
@@ -47,49 +84,112 @@ dev
...
+Fuji
+gamification
+Ganache
+getOwner
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:107]: is this code?

File: .github/styles/Vocab/Products/accept.txt [hunk 13]
```diff
@@ -47,49 +84,112 @@ dev
...
+getters
+gif
 GitHub
 github
 Golang
 gwei
 Gwei
+Hardforks
+Harpalsinh
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:116]: what is this word?

File: .github/styles/Vocab/Products/accept.txt [hunk 14]
```diff
@@ -47,49 +84,112 @@ dev
...
+hdwallet
+HDWallet
+i.e.
 incentivized
+Infura
+ipfs
+IPFS
 IPs
+isRegistered
+Jadeja
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:126]: Is this a word?

File: .github/styles/Vocab/Products/accept.txt [hunk 15]
```diff
@@ -47,49 +84,112 @@ dev
...
+Harpalsinh
+hdwallet
+HDWallet
+i.e.
 incentivized
+Infura
+ipfs
+IPFS
 IPs
+isRegistered
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:125]: Is this in code?

File: .github/styles/Vocab/Products/accept.txt [hunk 16]
```diff
@@ -47,49 +84,112 @@ dev
...
+Jadeja
 JavaScript
+jpg
 json
 JSON
+Keccak
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:131]: Is this a word?

File: .github/styles/Vocab/Products/accept.txt [hunk 17]
```diff
@@ -47,49 +84,112 @@ dev
...
 keyName
+keystore
 KYCed
 liveness
+lol
 lookups
+mainnet
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:138]: Same with Fuji, mainnet should have first letter upper-case.

File: .github/styles/Vocab/Products/accept.txt [hunk 18]
```diff
@@ -47,49 +84,112 @@ dev
...
 Mainnet
+Manov
 Markdownlint
 mempool
 Mempool
+metamask
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:144]: No lower case of this word

File: .github/styles/Vocab/Products/accept.txt [hunk 19]
```diff
@@ -47,49 +84,112 @@ dev
...
 MetaMask
+Metodi
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:146]: Is this a word?

File: .github/styles/Vocab/Products/accept.txt [hunk 20]
```diff
@@ -47,49 +84,112 @@ dev
...
+minimalistic
 mintable
 Mintable
+multicurrency
 multisig
 multistep
 Nakamoto
+navbar
+NPM
+nft
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:156]: no lower case

File: .github/styles/Vocab/Products/accept.txt [hunk 21]
```diff
@@ -47,49 +84,112 @@ dev
...
+NFT
+NFTs
+Nimish
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:159]: Is this a word?

File: .github/styles/Vocab/Products/accept.txt [hunk 22]
```diff
@@ -47,49 +84,112 @@ dev
...
 npm
+npx
+Ofem
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:162]: is this a word?

File: .github/styles/Vocab/Products/accept.txt [hunk 23]
```diff
@@ -47,49 +84,112 @@ dev
...
 OK
+onboarding
+O'Reilly
+Osutuk
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:166]: Is this a word?

File: .github/styles/Vocab/Products/accept.txt (hunk 21 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:159]: This is a person's name

File: .github/styles/Vocab/Products/accept.txt [hunk 24]
```diff
@@ -47,49 +84,112 @@ dev
...
+Pangolin's
 parameterizable
 parameterization
+params
 performant
 permissioned
 permissionless
+png
+PNG
+polymod
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:178]: not sure about this word

File: .github/styles/Vocab/Products/accept.txt (hunk 22 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:162]: This is a person's name

File: .github/styles/Vocab/Products/accept.txt [hunk 25]
```diff
@@ -47,49 +84,112 @@ dev
...
+postgres
+Postgres
 precompile
 Precompile
 precompiles
 Precompiles
+prefixChk
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:185]: This is not a word

File: .github/styles/Vocab/Products/accept.txt (hunk 19 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:146]: This is a person's name

File: .github/styles/Vocab/Products/accept.txt [hunk 26]
```diff
@@ -47,49 +84,112 @@ dev
...
+Keccak
 keyName
+keystore
 KYCed
 liveness
+lol
 lookups
+mainnet
 Mainnet
+Manov
```
[the remark this answers is not in the document]
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:140]: This is a person's name

File: .github/styles/Vocab/Products/accept.txt [hunk 27]
```diff
@@ -47,49 +84,112 @@ dev
...
 precompiles
 Precompiles
+prefixChk
 prefunded
 Proto
+Ranjan
+React's
 reCAPTCHA
+recoverability
+recoverAddress
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:192]: Is this in code?

File: .github/styles/Vocab/Products/accept.txt [hunk 28]
```diff
@@ -98,31 +198,81 @@ relayer
 Relayer
 relayers
 Relayers
+remix's
+renounceOwnership
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:202]: Is this in code?

File: .github/styles/Vocab/Products/accept.txt (hunk 14 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:126]: This is a person's name

File: .github/styles/Vocab/Products/accept.txt (hunk 13 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:116]: This is a person's name

File: .github/styles/Vocab/Products/accept.txt (hunk 28 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:202]: Yes. It's a variable name which is called out in a markdown file.

File: .github/styles/Vocab/Products/accept.txt (hunk 25 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:185]: It's a variable name which is called out in a markdown file.

File: .github/styles/Vocab/Products/accept.txt [hunk 29]
```diff
@@ -98,31 +198,81 @@ relayer
...
+Repo
+requesters
+Rimble
 RPC
 RPCs
+runlog
+RunLog
+Saglam
+Sayan
+secondBuyer
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:213]: should this be `second buyer`?

File: .github/styles/Vocab/Products/accept.txt (hunk 24 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:178]: It's a variable name which is called out in a markdown file.

File: .github/styles/Vocab/Products/accept.txt (hunk 23 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:166]: It's a person's name

File: .github/styles/Vocab/Products/accept.txt (hunk 16 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:131]: Yes it's a hashing algorithm.

File: .github/styles/Vocab/Products/accept.txt [hunk 30]
```diff
@@ -98,31 +198,81 @@ relayer
...
 subsampling
 Subsampling
+subtree
+sudo
 superset
+Swapper's
+testnet
 Testnet
 textlint
+Trufflesuite
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:240]: Truffle suite?

File: .github/styles/Vocab/Products/accept.txt (hunk 15 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:125]: Yes. It's a variable name which is called out in a markdown file.

File: .github/styles/Vocab/Products/accept.txt [hunk 31]
```diff
@@ -98,31 +198,81 @@ relayer
...
 TXs
+typechain
+TypeChain
+typings
+UIs
+unicode
+Uniswap
+unitconverter
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:248]: unit converter?

File: .github/styles/Vocab/Products/accept.txt (hunk 10 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:100]: It's a variable name which is called out in a markdown file.

File: .github/styles/Vocab/Products/accept.txt (hunk 9 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:98]: Yes. It's a variable name which is called out in a markdown file.

File: .github/styles/Vocab/Products/accept.txt (hunk 8 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:87]: It's a variable name which is called out in a markdown file.

File: .github/styles/Vocab/Products/accept.txt (hunk 29 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:213]: It's a variable name which is called out in a markdown file.

File: .github/styles/Vocab/Products/accept.txt [hunk 32]
```diff
@@ -98,31 +198,81 @@ relayer
...
+repo
+Repo
+requesters
+Rimble
 RPC
 RPCs
+runlog
+RunLog
+Saglam
+Sayan
```
[the remark this answers is not in the document]
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:212]: They are people's names

File: .github/styles/Vocab/Products/accept.txt (hunk 4 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:35]: I don't see that. Which PR do you mean?

File: .github/styles/Vocab/Products/accept.txt (hunk 7 above)
Comment (dasconnor) [reply .github/styles/Vocab/Products/accept.txt:74]: Should always be `dapp` unless in title case at the beginning of a sentence.

Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:74]: That's what we're doing.

File: .github/styles/Vocab/Products/accept.txt (hunk 4 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:35]: I see it here. https://github.com/ava-labs/avalanche-docs/pull/1115

Removing from `accept.txt` and resolving this comment.

File: .github/styles/Vocab/Products/accept.txt (hunk 5 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:7]: Confirmed

File: .github/styles/Vocab/Products/accept.txt (hunk 8 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 10 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 12 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 15 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 24 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 27 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 28 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 29 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 31 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 25 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 9 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 6 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 7 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt:65]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 11 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 17 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 18 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 20 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 30 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt [hunk 33]
```diff
@@ -1,44 +1,72 @@
...
+crypto
+cryptocurrencies
+cryptocurrency
+Ctrl
+Daly
+DAO
 dapp
+Dapp
 delegators
+deploy_f
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:67]: Can you check on this?

File: .github/styles/Vocab/Products/accept.txt [hunk 34]
```diff
@@ -48,48 +76,99 @@ Dexalot
...
 GitHub
 github
 Golang
 gwei
 Gwei
+Hardforks
+Harpalsinh
+hdwallet
+HDWallet
+i.e.
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:105]: Is this needed?

File: .github/styles/Vocab/Products/accept.txt [hunk 35]
```diff
@@ -48,48 +76,99 @@ Dexalot
...
+Pangolin's
 parameterizable
 parameterization
+params
 performant
 permissioned
 permissionless
+png
+PNG
+postgres
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:160]: Probably remove this?

File: .github/styles/Vocab/Products/accept.txt [hunk 36]
```diff
@@ -98,31 +177,77 @@ relayer
...
+Repo
+requesters
+Rimble
 RPC
 RPCs
+runlog
+RunLog
+Saglam
+Sayan
+setOwner
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:191]: Is this in code?

File: .github/styles/Vocab/Products/accept.txt [hunk 37]
```diff
@@ -98,31 +177,77 @@ relayer
...
+snowtrace
```
Comment (yulin-dong) [inline .github/styles/Vocab/Products/accept.txt:192]: Remove this and always use Snowtrace?

File: .github/styles/Vocab/Products/accept.txt (hunk 33 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 35 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 37 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 36 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 34 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

**Review by yulin-dong — APPROVED** [review]

**Pull request merged**